app/test/frontend/golden/analysis_tab_http.html

```diff
@@ -78,49 +78,83 @@ <h4>Platforms</h4>
+    <span class="suggestion-icon-danger"></span>
+    <p>Running <code>dartdoc</code> failed.</p>
+
+  </div>
+  <div class="suggestion-description">
+    <p>Make sure <code>dartdoc</code> runs without any issues.</p>
+
+    
+  </div>
 </div>
-      <blockquote><p>Analysis or formatting checks reported 37 hints.</p>
+<div class="suggestion-row">
+  <div class="suggestion-title">
+    <span class="suggestion-icon-info"></span>
+    <p>Fix analysis and formatting issues.</p>
+
+  </div>
+  <div class="suggestion-description">
+    <p>Analysis or formatting checks reported 37 hints.</p>
 <p>Run <code>dartfmt</code> to format <code>lib/browser_client.dart</code>.</p>
 <p>Strong-mode analysis of <code>lib/http.dart</code> gave the following hint:</p>
 <p>line: 88 col: 29<br />
 'UTF8' is deprecated and shouldn't be used.</p>
 <p>Similar analysis of the following files failed:</p><ul><li><code>lib/src/base_client.dart</code> (hint)</li><li><code>lib/src/base_request.dart</code> (hint)</li><li><code>lib/src/base_response.dart</code> (hint)</li><li><code>lib/src/byte_stream.dart</code> (hint)</li><li><code>lib/src/client.dart</code> (hint)</li><li><code>lib/src/io_client.dart</code> (hint)</li><li><code>lib/src/mock_client.dart</code> (hint)</li><li><code>lib/src/multipart_file.dart</code> (hint)</li><li><code>lib/src/multipart_request.dart</code> (hint)</li><li><code>lib/src/request.dart</code> (hint)</li><li><code>lib/src/response.dart</code> (hint)</li><li><code>lib/src/streamed_request.dart</code> (hint)</li><li><code>lib/src/streamed_response.dart</code> (hint)</li><li><code>lib/src/utils.dart</code> (hint)</li></ul>
-</blockquote>
-    </li>
-    <li>
-      <div><p>The description is too short.</p>
+
+    
+  </div>
 </div>
-      <blockquote><p>Add more detail about the package, what it does and what is its target use case. Try to write at least 60 characters.</p>
-</blockquote>
-    </li>
-    <li>
-      <div><p>Package is pre-v1 release.</p>
+<div class="suggestion-row">
+  <div class="suggestion-title">
+    <span class="suggestion-icon-info"></span>
+    <p>The description is too short.</p>
+
+  </div>
+  <div class="suggestion-description">
+    <p>Add more detail about the package, what it does and what is its target use case. Try to write at least 60 characters.</p>
+
+    
+  </div>
 </div>
-      <blockquote><p>While there is nothing inherently wrong with versions of <code>0.*.*</code>, it usually means that the author is still experimenting with the general direction API.</p>
-</blockquote>
-    </li>
-    <li>
-      <div><p>Maintain an example.</p>
+<div class="suggestion-row">
+  <div class="suggestion-title">
+    <span class="suggestion-icon-info"></span>
+    <p>Package is pre-v1 release.</p>
+
+  </div>
+  <div class="suggestion-description">
+    <p>While there is nothing inherently wrong with versions of <code>0.*.*</code>, it usually means that the author is still experimenting with the general direction API.</p>
```

Comment:
s/general direction API/general direction of the API/g?

Reply:
Coming from pana, created an issue for it: https://github.com/dart-lang/pana/issues/360